examples/runner-default/jon.tfvars

```diff
@@ -0,0 +1 @@
+registration_token = "GR1348941EEPc63Lgud5D3DReD4nM"
```

Comment:
guess you committed this file accidentally

Reply:
indeed, I'll remove that now!

Reply:
Done @npalm . Apologies for the long delay on this. I just noticed I am still using a fork of this module,  and keen to resolve.